src/test/java/uk/gov/ch/oauth/identity/IdentityProviderTests.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.ch.oauth.identity;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+import static org.junit.Assert.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class IdentityProviderTests {
+
+
+    private IdentityProvider provider;
+
+    private EnvironmentReaderImpl reader;
+    private  String code= "code";
+    private  String state = "state";
+    private  String scope = "scope";
```

Comment:
Could you change these into static final constants.

Reply:
Done :)

Reply:
Typically we write constants variable names in all caps.

Reply:
Sorry, I forgot to change that back on Friday I will change that now.